Week1/homework/world-queries.txt

```diff
@@ -37,4 +37,181 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
 | Heerlen           |
 | Alkmaar           |
 +-------------------+
-28 rows in set (0.00 sec)
\ No newline at end of file
+28 rows in set (0.00 sec)
+
+2. What is the population of Rotterdam ?
+
+select Population from city where city.Name="Rotterdam";
```

Comment:
:+1:

Remember to use `SELECT` in all-caps instead of `select` - it executes the same (the DBMS doesn't care) but it makes it much easier for a reader of your code to distinguish between keywords and column/table names.